hedera-mirror-rest/tokens.js

```diff
@@ -397,45 +397,47 @@ const transformTimestampFilterOp = (op) => {
  * @return {{query: string, params: []}} the query string and params
  */
 const extractSqlFromTokenInfoRequest = (tokenId, filters) => {
-  const conditions = [`${CustomFee.TOKEN_ID} = $1`];
+  const conditions = [`${CustomFee.getFullName(CustomFee.TOKEN_ID)} = $1`];
   const params = [tokenId];
 
+  let customFeeQuery = `
+        ${customFeeSelect}
+        from ${CustomFee.tableName} ${CustomFee.tableAlias}
+        where ${conditions.join(' and ')}
+        order by ${CustomFee.getFullName(CustomFee.CREATED_TIMESTAMP)} desc`;
   if (filters && filters.length !== 0) {
     // honor the last timestamp filter
     const filter = filters[filters.length - 1];
     const op = transformTimestampFilterOp(filter.operator);
-    conditions.push(`${CustomFee.FILTER_MAP[filter.key]} ${op} $2`);
+    conditions.push(`lower(${CustomFee.FILTER_MAP[filter.key]}) ${op} $2`);
```

Comment:
perhaps just remove the `FILTER_MAP` in the model class since there is just one, then change it to
 ```suggestion
    conditions.push(`lower(${CustomFee.TIMESTAMP_RANGE}) ${op} $2`);
```